lib/driver.js

```diff
@@ -42,8 +42,11 @@ class AndroidDriver extends BaseDriver {
     this.sessionChromedrivers = {};
     this.jwpProxyActive = false;
     this.jwpProxyAvoid = _.clone(NO_PROXY);
-    this.settings = new DeviceSettings({ignoreUnimportantViews: false},
-                                       this.onSettingsUpdate.bind(this));
+    this.settings = new DeviceSettings({ignoreUnimportantViews: false,
```

Comment:
this needs a little formatting work. first, let's do one key/value per line. second, we do `key: value` (with a space after colon. third, we do:

```js
foo({
  new: 'line',
  before: 'keys/values'
});
```

to save horizontal space :-)